match/intGreaterThanOrEqualTo.go

```diff
@@ -0,0 +1,62 @@
+package match
+
+import (
+	"reflect"
+)
+
+// IntGreaterThanOrEqualTo returns a new matcher that will match int's greater than or equal to the provided int
+func IntGreaterThanOrEqualTo(value int) SupportedKindsMatcher {
```

Comment:
use int64 for the value to prevent issues on 32-bit systems where `int` is likely `int32`?